sdk/storage/inc/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -639,77 +644,69 @@ namespace Azure { namespace Storage { namespace DataLake {
   {
     std::unique_ptr<Azure::Core::Http::BodyStream> BodyStream;
     std::string AcceptRanges;
-    std::string CacheControl;
-    std::string ContentDisposition;
-    std::string ContentEncoding;
-    std::string ContentLanguage;
+    DataLakeHttpHeaders HttpHeaders;
     int64_t ContentLength = int64_t();
-    std::string ContentRange;
-    std::string ContentType;
-    std::string ContentMD5;
+    Azure::Core::Nullable<std::string> ContentRange;
+    Azure::Core::Nullable<std::string> ContentMD5;
     std::string Date;
     std::string ETag;
     std::string LastModified;
     std::string RequestId;
     std::string Version;
     std::string ResourceType;
-    std::string Properties;
-    std::string LeaseDuration;
-    std::string LeaseState;
-    std::string LeaseStatus;
-    std::string XMsContentMd5;
+    Azure::Core::Nullable<std::string> Properties;
+    Azure::Core::Nullable<std::string> LeaseDuration;
+    Azure::Core::Nullable<std::string> LeaseState;
+    Azure::Core::Nullable<std::string> LeaseStatus;
+    Azure::Core::Nullable<std::string> XMsContentMd5;
```

Comment:
This isn't a good name

Reply:
Refined this to `ContentMd5` and the other to `TransactionalMd5`.